source/_code.html.erb

```diff
@@ -0,0 +1,14 @@
+<div class="container">
+  <button aria-controls="<%= name %>-snippets"
+          aria-expanded="false"
```

Comment:
Makes sense! This is exactly the type of project to split hairs on, and I appreciate it. I'll give this a go